src/relay/pass/alpha_eq.cc

```diff
@@ -0,0 +1,259 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file src/tvm/relay/pass/AlphaEqual.cc
+ * \brief Compute the set of variables not bound in the expression.
+ */
+#include <tvm/relay/expr_visitor.h>
+#include "tvm/relay/pass/alpha_eq.h"
+#include "./type_visitor.h"
+
+namespace tvm {
+namespace relay {
+
+using namespace tvm::runtime;
+
+struct TypeAlphaEq : TypeVisitor<const Type &> {
```

Comment:
const Type&

I would recommend we use some code format tools to format the code, because I've seen inconsistent position of & and * in many files.

Reply:
Yeah this is due to a few different people having written chunks to this before we released the alpha. I have been using clang-format but there are issues with it violating the style used by TVM. It would be great to commit a single clang-format configuration file we can all share.

Reply:
It seems that under the Google format `clang-format` will not change the position of `&`.

Reply:
I should be able to do that because I am using it. Are you using IDE or VIM?